A few LoggingSpec tests are failing

A few LoggingSpec tests are failing to me on a freshly cloned project:

This is Windows 10 x64, Oracle Corporation Java 1.8.0_181, tests run from IntelliJ sbt shell.

Running tests from the IDE 

```
-------------------------------- Running Tests --------------------------------
[info]
X io.bullet.borer.LoggingSpec.simple values 275ms


  java.lang.AssertionError: assertion failed: ==> assertion failed: 1: [
       1/11: null
       2/11: undefined
       3/11: true
       4/11: 42
       5/11: 2147483648L
       6/11: 0xffffffffffffffff
       7/11: -0xffffffffffffffff
       8/11: 1.0f16
       9/11: 100000.0f
      10/11: 1.6
      11/11: SimpleValue(18)
  1: ]
  2: END
   != 1: [
       1/11: null
       2/11: undefined
       3/11: true
       4/11: 42
       5/11: 2147483648L
       6/11: 0xffffffffffffffff
       7/11: -0xffffffffffffffff
       8/11: 1.0f16
       9/11: 100000.0f
      10/11: 1.6
      11/11: SimpleValue(18)
  1: ]
  2: END
    scala.Predef$.assert(Predef.scala:282)
    utest.asserts.Asserts$ArrowAssert.$eq$eq$greater(Asserts.scala:90)
    io.bullet.borer.LoggingSpec$.roundTripLogEquals(LoggingSpec.scala:221)
    io.bullet.borer.LoggingSpec$.$anonfun$tests$2(LoggingSpec.scala:34)
[info] No tests to run for compat-circe / Test / testQuick
X io.bullet.borer.LoggingSpec.byte arrays 8ms
  java.lang.AssertionError

  java.lang.AssertionError: assertion failed: ==> assertion failed: 1: [
      1/3: BYTES[41 42 43 44 45 46 47 48]
      2/3: BYTES[41 42 43 44 45 46 47 48 ...]
      3/3: BYTES-STREAM[
          1: BYTES[41 42 43 44]
          2: BYTES[45 46 47 48]
          3: BYTES[49 4A 4B 4C]



          4: BYTES[4D 4E 4F 50]
          5: BYTES[51 52 53 54]
          6: BYTES[55 56 57 58]
          7: BYTES[59 5A]
      3/3: ]
  1: ]
  2: END
   != 1: [
      1/3: BYTES[41 42 43 44 45 46 47 48]
      2/3: BYTES[41 42 43 44 45 46 47 48 ...]
      3/3: BYTES-STREAM[
          1: BYTES[41 42 43 44]
          2: BYTES[45 46 47 48]
          3: BYTES[49 4A 4B 4C]
          4: BYTES[4D 4E 4F 50]
          5: BYTES[51 52 53 54]
          6: BYTES[55 56 57 58]
          7: BYTES[59 5A]
      3/3: ]
  1: ]
  2: END
    scala.Predef$.assert(Predef.scala:282)
    utest.asserts.Asserts$ArrowAssert.$eq$eq$greater(Asserts.scala:90)
    io.bullet.borer.LoggingSpec$.roundTripLogEquals(LoggingSpec.scala:221)
    io.bullet.borer.LoggingSpec$.$anonfun$tests$3(LoggingSpec.scala:57)
[info]
[info]
[info] No tests to run for compat-scodec / Test / testQuick
X io.bullet.borer.LoggingSpec.text arrays 137ms 
  java.lang.AssertionError: assertion failed: ==> assertion failed: 1: [
      1/3: "ABCDEFGH"
      2/3: "ABCDEFGH..."
      3/3: TEXT-STREAM[
          1: "ABCD"
          2: "EFGH"
          3: "IJKL"
          4: "MNOP"
          5: "QRST"
          6: "UVWX"
          7: "YZ"
      3/3: ]
  1: ]
  2: END
   != 1: [
      1/3: "ABCDEFGH"
      2/3: "ABCDEFGH..."
      3/3: TEXT-STREAM[
          1: "ABCD"
          2: "EFGH"
          3: "IJKL"
          4: "MNOP"
          5: "QRST"
          6: "UVWX"
          7: "YZ"
      3/3: ]
  1: ]
  2: END
    scala.Predef$.assert(Predef.scala:282)
    utest.asserts.Asserts$ArrowAssert.$eq$eq$greater(Asserts.scala:90)
    io.bullet.borer.LoggingSpec$.roundTripLogEquals(LoggingSpec.scala:221)
    io.bullet.borer.LoggingSpec$.$anonfun$tests$5(LoggingSpec.scala:80)
[info] No tests to run for compat-cats / Test / testQuick
[info]
[info] No tests to run for derivation / Test / testQuick
X io.bullet.borer.LoggingSpec.arrays 1ms 
  java.lang.AssertionError: assertion failed: ==> assertion failed: 1: [
      1/3: [
          1/3: 1
          2/3: 2
          3/3: 3
      1/3: ]
      2/3: []
      3/3: [
          1: 1
          2: 2
          3: 3
      3/3: ]
  1: ]
  2: END
   != 1: [
      1/3: [
          1/3: 1
          2/3: 2
          3/3: 3
      1/3: ]
      2/3: []
      3/3: [
          1: 1
          2: 2
          3: 3
      3/3: ]
  1: ]
  2: END
    scala.Predef$.assert(Predef.scala:282)
    utest.asserts.Asserts$ArrowAssert.$eq$eq$greater(Asserts.scala:90)
    io.bullet.borer.LoggingSpec$.roundTripLogEquals(LoggingSpec.scala:221)
    io.bullet.borer.LoggingSpec$.$anonfun$tests$6(LoggingSpec.scala:103)
[info]
[info]
[info] No tests to run for compat-akka / Test / testQuick
[info] No tests to run for site / Test / testQuick
X io.bullet.borer.LoggingSpec.maps 13ms 
  java.lang.AssertionError: assertion failed: ==> assertion failed: 1: {
      1/3: "foo"
      1/3: -> 42
      2/3: "empty"
      2/3: -> {}
      3/3: "bar"
      3/3: -> {
          1: "a"
          1: -> "a"
          2: "b"
          2: -> "b"
          3: "c"
          3: -> "c"
      3/3: }
  1: }
  2: END
   != 1: {
      1/3: "foo"
      1/3: -> 42
      2/3: "empty"
      2/3: -> {}
      3/3: "bar"
      3/3: -> {
          1: "a"
          1: -> "a"
          2: "b"
          2: -> "b"
          3: "c"
          3: -> "c"
      3/3: }
  1: }
  2: END
    scala.Predef$.assert(Predef.scala:282)
    utest.asserts.Asserts$ArrowAssert.$eq$eq$greater(Asserts.scala:90)
    io.bullet.borer.LoggingSpec$.roundTripLogEquals(LoggingSpec.scala:221)
    io.bullet.borer.LoggingSpec$.$anonfun$tests$7(LoggingSpec.scala:127)
[info] Passed: Total 0, Failed 0, Errors 0, Passed 0
X io.bullet.borer.LoggingSpec.long arrays and maps 5ms 
  java.lang.AssertionError: assertion failed: ==> assertion failed: 1: [
      1/2: [
           1/21: 0
           2/21: 1
           3/21: 2
           4/21: 3
           5/21: 4
           6/21: 5
          ...
          16/21: 15
          17/21: 16
          18/21: 17
          19/21: 18
          20/21: 19
          21/21: 20
      1/2: ]
      2/2: {
           1/21: 0
           1/21: -> "A"
           2/21: 1
           2/21: -> "B"
           3/21: 2
           3/21: -> "C"
           4/21: 3
           4/21: -> "D"
           5/21: 4
           5/21: -> "E"
           6/21: 5
           6/21: -> "F"
           7/21: 6
           7/21: -> "G"
           8/21: 7
           8/21: -> "H"
          ...
          15/21: 14
          15/21: -> "O"
          16/21: 15
          16/21: -> "P"
          17/21: 16
          17/21: -> "Q"
          18/21: 17
          18/21: -> "R"
          19/21: 18
          19/21: -> "S"
          20/21: 19
          20/21: -> "T"
          21/21: 20
          21/21: -> "U"
      2/2: }
  1: ]
  2: END
   != 1: [
      1/2: [
           1/21: 0
           2/21: 1
           3/21: 2
           4/21: 3
           5/21: 4
           6/21: 5
          ...
          16/21: 15
          17/21: 16
          18/21: 17
          19/21: 18
          20/21: 19
          21/21: 20
      1/2: ]
      2/2: {
           1/21: 0
           1/21: -> "A"
           2/21: 1
           2/21: -> "B"
           3/21: 2
           3/21: -> "C"
           4/21: 3
           4/21: -> "D"
           5/21: 4
           5/21: -> "E"
           6/21: 5
           6/21: -> "F"
           7/21: 6
           7/21: -> "G"
           8/21: 7
           8/21: -> "H"
          ...
          15/21: 14
          15/21: -> "O"
          16/21: 15
          16/21: -> "P"
          17/21: 16
          17/21: -> "Q"
          18/21: 17
          18/21: -> "R"
          19/21: 18
          19/21: -> "S"
          20/21: 19
          20/21: -> "T"
          21/21: 20
          21/21: -> "U"
      2/2: }
  1: ]
  2: END
    scala.Predef$.assert(Predef.scala:282)
    utest.asserts.Asserts$ArrowAssert.$eq$eq$greater(Asserts.scala:90)
    io.bullet.borer.LoggingSpec$.roundTripLogEquals(LoggingSpec.scala:221)
    io.bullet.borer.LoggingSpec$.$anonfun$tests$9(LoggingSpec.scala:152)
[info] No tests to run for benchmarks / Test / testQuick
[info] Tests: 6, Passed: 0, Failed: 6
[error] Failed tests:
[error]         io.bullet.borer.LoggingSpec
[info]
[info]
[info]
[info] No tests to run for compat-scodecJS / Test / testQuick
[info] No tests to run for derivationJS / Test / testQuick
[info] No tests to run for coreJS / Test / testQuick
[info] utest
[info]
[info] Passed: Total 0, Failed 0, Errors 0, Passed 0
[info] No tests to run for compat-circeJS / Test / testQuick
[info]
[info] No tests to run for compat-catsJS / Test / testQuick
[error] (core / Test / testQuick) sbt.TestsFailedException: Tests unsuccessful
[error] Total time: 9 s, completed 11.11.2020 12:55:34
```

--- a/core/src/test/scala/io/bullet/borer/LoggingSpec.scala
+++ b/core/src/test/scala/io/bullet/borer/LoggingSpec.scala
@@ -215,7 +215,8 @@ object LoggingSpec extends TestSuite {
           maxShownByteArrayPrefixLen = 8,
           maxShownStringPrefixLen = 8,
           maxShownArrayElems = 12,
-          maxShownMapEntries = 15)
+          maxShownMapEntries = 15,
+          lineSeparator = "\n")
         .toByteArray
 
     log.toString ==> expectedLog
@@ -228,7 +229,8 @@ object LoggingSpec extends TestSuite {
         maxShownByteArrayPrefixLen = 8,
         maxShownStringPrefixLen = 8,
         maxShownArrayElems = 12,
-        maxShownMapEntries = 15)
+        maxShownMapEntries = 15,
+        lineSeparator = "\n")
       .to[Element]
       .value ==> element
 

